$OpenBSD: patch-modules_codec_avcodec_avcodec_c,v 1.2 2013/03/30 02:06:26 brad Exp $

Re-enable the multi-threaded mode for the avcodec backend.

--- modules/codec/avcodec/avcodec.c.orig	Tue Mar 19 19:45:29 2013
+++ modules/codec/avcodec/avcodec.c	Tue Mar 19 19:46:39 2013
@@ -132,7 +132,7 @@ vlc_module_begin ()
     add_bool( "ffmpeg-hw", false, HW_TEXT, HW_LONGTEXT, false )
 #endif
 #if defined(FF_THREAD_FRAME)
-    add_integer( "ffmpeg-threads", 1, THREADS_TEXT, THREADS_LONGTEXT, true );
+    add_integer( "ffmpeg-threads", 0, THREADS_TEXT, THREADS_LONGTEXT, true );
 #endif
 
 
